src/lib.rs

```diff
@@ -339,7 +339,7 @@ fn is_nontranslatable_codeblock_group(events: &[(usize, Event)]) -> bool {
             // Heuristic to check whether the codeblock nether has a
             // literal string nor a line comment.  We may actually
             // want to use a lexer here to make this more robust.
-            !codeblock_text.contains("\"") && !codeblock_text.contains("//")
+            !codeblock_text.contains('"') && !codeblock_text.contains("//")
```

Comment:
Looks like a good, but unrelated change — please submit it in a separate PR and have @dyoo approve it :smile:

Reply:
Thanks for #85.